src/pytorch_lightning/strategies/colossalai.py

```diff
@@ -0,0 +1,275 @@
+from typing import Any, Generator, Optional
+
+import torch
+
+import pytorch_lightning as pl
+from pytorch_lightning.accelerators.cuda import CUDAAccelerator
+from pytorch_lightning.overrides.base import _LightningModuleWrapperBase, _LightningPrecisionModuleWrapperBase
+from pytorch_lightning.plugins.precision import ColossalAIPrecisionPlugin
+from pytorch_lightning.strategies.ddp import DDPStrategy
+from pytorch_lightning.utilities.exceptions import MisconfigurationException
+from pytorch_lightning.utilities.imports import _RequirementAvailable
+from pytorch_lightning.utilities.types import STEP_OUTPUT
+
+_COLOSSALAI_AVAILABLE = _RequirementAvailable("colossalai")
```

Comment:
```suggestion
from lightning_utilities.core.imports import RequirementCache
from pytorch_lightning.utilities.types import STEP_OUTPUT

_COLOSSALAI_AVAILABLE = RequirementCache("colossalai")
```

Reply:
this will fix the CI and run tests

Reply:
Thanks. I've corrected it.

Reply:
thanks! looks like there are a few more utilities that were moved to `lightning_utilities.core` recently that is causing this issue and blocking ci tests. Can you try running them locally using PL master and updating the imports? I can't do it since I don't have edit access to their branch.

Reply:
I have tried run tests locally, but I failed. I just set the environment variable `PL_RUN_STANDALONE_TESTS` to 1 and directly used command `pytest` to run all the tests. But the test went endlessly. I wonder what is the correct way to run the tests.

Reply:
this is the correct way to run them, but for now the issues are coming from import so you can run the ones that are not standalone and we can later check the CI with correct imports.